src/api/libcellml/types.h

```diff
@@ -44,7 +48,7 @@ using ValidatorPtr = std::shared_ptr<Validator>; /**< Type definition for shared
 
 // CellML entities.
 class Component; /**< Forward declaration of Component class. */
-using ComponentPtr = std::shared_ptr<Component>; /**< Type definition for shared component pointer. */
+using ComponentPtr = std::shared_ptr<Component>; /**< Type definition for shared component pointer. @details Some extra definition for detailed description. */
```

Comment:
... is this you just playing with doxygen translation stuff ... ?

Reply:
Yes.  It should all be removed.